SocialStudies/SurviveTheMiddleAges/survive-the-middle-ages.ipynb

```diff
@@ -0,0 +1,377 @@
+    "3. Run the birth cell for every married female over 12 years old.\n",
+    "4. If the year is between 1347 and 1351, you will also need to run the **plague** code cell."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Mortality\n",
+    "\n",
+    "In the next cell, set your character's age (e.g. `character_age = 1` for the second round) then run the cell to see if your character lives or dies, based on these approximate mortality rates.\n",
+    "\n",
+    "|Stage of Life|Age Range|Mortality Rate|\n",
+    "|---|---|---|\n",
+    "|Infancy|0-3|30%-50%|\n",
+    "|Childhood|4-11|20%|\n",
+    "|Young Adulthood|12-21|10%|\n",
+    "|Adulthood|22-49|10%|\n",
+    "|Elder|50+|30%|"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "character_age = 0\n",
+    "\n",
+    "print('My character is', character_age, 'years old.')\n",
+    "if character_age < 4:\n",
+    "    mortality_rate = 30\n",
+    "elif character_age < 12:\n",
+    "    mortality_rate = 20\n",
+    "elif character_age < 50:\n",
+    "    mortality_rate = 10\n",
+    "else:\n",
+    "    mortality_rate = 30\n",
+    "mortality_probablity = ['die']*mortality_rate + ['live']*(100-mortality_rate)\n",
+    "live_die = random.choice(mortality_probablity)\n",
+    "if live_die == 'die':\n",
+    "    print('Unfortuately my character has died.')\n",
+    "if live_die == 'live':\n",
+    "    print('My character has survived the year and is now', character_age+1, 'years old.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Marriage\n",
+    "\n",
+    "If your character is not already married, set `character_age` and `character_gender` then run the cell to see if your character gets married or not, based on these approximate marriageability rates.\n",
+    "\n",
+    "|Stage of Life|Age Range|Marriageability Rate|\n",
+    "|---|---|---|\n",
+    "|Noble Infant|0-3|10%|\n",
+    "|Noble Child|4-11|10%|\n",
+    "|Noble Young Adult Male|12-21|30%|\n",
+    "|Noble Young Adult Female|12-21|90%|\n",
+    "|Noble Adult Male|22-49|60%|\n",
+    "|Noble Adult Female|22-49|70%|\n",
+    "|Noble Elder Male|50+|90%|\n",
+    "|Noble Elder Female|50+|70%|\n",
+    "|Infant|0-3|0%|\n",
+    "|Child|4-11|0%|\n",
+    "|Young Adult Male|12-21|30%|\n",
+    "|Young Adult Female|12-21|50%|\n",
+    "|Adult Male|22-49|60%|\n",
+    "|Adult Female|22-49|80%|\n",
+    "|Elder Male|50+|80%|\n",
+    "|Elder Female|50+|10%|\n",
+    "\n",
+    "If your character gets married, add a spouse to your character's family. You can decide on the age of the spouse."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "character_age = 2\n",
+    "character_gender = 'female'\n",
+    "\n",
+    "if character_class == 'Noble':\n",
```

Comment:
Although I think implementing a [switch-case](https://data-flair.training/blogs/python-switch-case/) is best practice in a scenario like this, I would at least try to reduce the redundancy in the code here by trying to just simply [check for multiple conditions](https://www.geeksforgeeks.org/check-multiple-conditions-in-if-statement-python/) as the bulkiness of the code segment here reduces the readability gained by nesting single conditional statements

Reply:
I like your idea about checking for multiple conditions, hopefully that makes it more readable. It is unfortunate that Python doesn't have a built-in switch-case, but I wanted to keep this code as approachable as possible.